addon-api/content-script/Tab.js

```diff
@@ -207,6 +207,7 @@ export default class Tab extends Listenable {
    * @type {?string}
    */
   get editorMode() {
+    if (location.origin === "https://scratchfoundation.github.io" || location.port === "8601") return "editor";
```

Comment:
From https://github.com/ScratchAddons/ScratchAddons/pull/6253#issuecomment-1603850164

This should have considered fullscreen mode.